front/app/containers/Admin/projects/project/events/components/EventMap.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { memo, useRef } from 'react';
+
+// components
+import EsriMap from 'components/EsriMap';
+import Graphic from '@arcgis/core/Graphic';
+import Point from '@arcgis/core/geometry/Point';
+import MapView from '@arcgis/core/views/MapView';
+
+// utils
+import { getMapPinSymbol } from 'components/EsriMap/utils';
+
+// types
+import { SubmitState } from '../edit';
+
+// hooks
+import { useTheme } from 'styled-components';
+
+export interface Props {
+  mapHeight?: string;
+  setSubmitState?: (state: SubmitState) => void;
+  position?: GeoJSON.Point | null;
+  setLocationPoint?: (locationPoint: GeoJSON.Point) => void;
+}
+
+const EventMap = memo<Props>(
+  ({ position, mapHeight, setLocationPoint, setSubmitState }: Props) => {
+    const theme = useTheme();
+    const locationPoint = useRef<GeoJSON.Point | null>(position || null);
```

Comment:
A ref is needed here so we don't get re-rendering of the map itself (which causes a brief white screen as the map re-loads).